yarn.lock

```diff
@@ -1451,6 +1451,11 @@
   resolved "https://registry.yarnpkg.com/@commitlint/types/-/types-11.0.0.tgz#719cf05fcc1abb6533610a2e0f5dd1e61eac14fe"
   integrity sha512-VoNqai1vR5anRF5Tuh/+SWDFk7xi7oMwHrHrbm1BprYXjB2RJsWLhUrStMssDxEl5lW/z3EUdg8RvH/IUBccSQ==
 
+"@contentful/app-sdk@^3.29.2":
```

Comment:
Looks like `ui-extensions-sdk` is used somewhere, otherwise I'd expect it to be removed from `yarn.lock`

Reply:
Never mind, just saw your push